go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/google/cel-go v0.15.1
 	github.com/google/gopacket v1.1.19
 	github.com/hashicorp/golang-lru v0.5.4
+	github.com/hashicorp/golang-lru/v2 v2.0.2
```

Comment:
can we relace all use of the `golang-lru` to `golang-lru/v2`? so we have everything on the latest version, instead of part using v2 and part using the older version?

Reply:
I think I did move all usages of `golang-lru` to `v2` but for some reason when `go mod tidy`ied afterwards it kept it there.